project/profiles/capacity4more/behat/features/bootstrap/FeatureContext/Group.php

```diff
@@ -698,4 +719,33 @@ public function iResetOrderToAlphabetical() {
     $el = $page->find('xpath', '//a[contains(text(),\'Order items alphabetically\') and not(ancestor::*[contains(@style,\'visibility: hidden\')])]');
     $el->click();
   }
+
+  /**
+   * @Then /^I should see the "([^"]*)" button$/
+   */
+  public function IShouldSeeTheButton($locator) {
```

Comment:
@ordavidil 
Was searching for it, but couldn't find anything like it.
There're functions that search for link /  text, but not for buttons.